ynr/apps/moderation_queue/views.py

```diff
@@ -214,7 +214,7 @@ def get_context_data(self, **kwargs):
             username = context["queued_image"].user.username
             email = context["queued_image"].user.email
         else:
-            username = "a robot 🤖"
```

Comment:
Maybe we should keep the `a robot 🤖` instead of just `bot` as that is what is used elsewhere on other pages? Also just because I like the emoji